workers/ohsome_quality_analyst/api.py

```diff
@@ -149,4 +154,53 @@ async def _fetch_report(
 
 @app.get("/regions")
 async def get_available_regions():
-    return await db_client.get_available_regions()
+    """List names of available regions."""
+    response = empty_api_response()
+    result = await db_client.get_available_regions()
+    response.update(result)
+    return response
+
+
+@app.get("/indicatorNames")
+async def list_indicators():
+    """List names of available indicators."""
+    response = empty_api_response()
+    response["result"] = list(load_metadata("indicators").keys())
+    return response
+
+
+@app.get("/datasetNames")
+async def list_datasets():
+    """List names of available datasets."""
+    response = empty_api_response()
+    response["result"] = list(list(DATASETS.keys()))
```

Comment:
Is it intentional that `list()` is called two times?

```suggestion
    response["result"] = list(DATASETS.keys())
```